data/templates/ocserv/radius_conf.tmpl

```diff
@@ -0,0 +1,26 @@
+### generated by cpn_anyconnect.py ###
+nas-identifier VyOS
+{% for srv in server %}
+{%   if not "disable" in server[srv] %}
+{%     if "port" in server[srv] %}
+authserver {{ srv }}:{{server[srv]["port"]}}
+{%     else %}
+authserver {{ srv }}
+{%     endif %}
+{%   endif %}
+{% endfor %}
+{% if timeout %}
+radius_timeout {{ timeout }}
```

Comment:
why not declaring `<defaultValue>2</defaultValue>` in the XML definition?

Reply:
Done